regression/goto-harness/chain.sh

```diff
@@ -25,16 +32,24 @@ else
   $goto_cc -o "$input_goto_binary" "$input_c_file"
 fi
 
-if [ -e "$harness_c_file" ] ; then
-  rm -f "$harness_c_file"
+if [ -e "$harness_file" ] ; then
+  rm -f "$harness_file"
 fi
 
 # `# some comment` is an inline comment - basically, cause bash to execute an empty command
 $cbmc --show-goto-functions "$input_goto_binary"
-$goto_harness "$input_goto_binary" "$harness_c_file" --harness-function-name $entry_point ${args}
-$cbmc --show-goto-functions "$harness_c_file"
-$cbmc --function $entry_point "$input_c_file" "$harness_c_file" \
+$goto_harness "$input_goto_binary" "$harness_file" --harness-function-name $entry_point ${args}
+$cbmc --show-goto-functions "$harness_file"
+if [[ "${harness_file}" == "harness.gb" ]];then
+  $cbmc --function $entry_point "$harness_file" \
+    --pointer-check `# because we want to see out of bounds errors` \
+    --unwind 11 `# with the way we set up arrays symex can't figure out loop bounds automatically` \
```

Comment:
Having these hardcoded in `chain.sh` I find worrying ... but I don't see and immediate alternative that is not a lot of extra complexity.

Reply:
Agreed, but I wanted to keep the changes to a minimum. The cleaner approach would be to include the output file name in each and every test of `goto-harness`.

Reply:
Fair.

Reply:
Its a little bit of a shame that the whole cbmc command line gets repeated here - but I guess the chain.sh is not likely to change too often, so the risk of adding options to only one of the two branches of the if is low.